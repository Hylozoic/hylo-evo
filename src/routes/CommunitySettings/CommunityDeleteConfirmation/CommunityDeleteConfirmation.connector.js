import { connect } from 'react-redux'
import { push } from 'connected-react-router'
import { allCommunitiesUrl } from 'util/navigation'

export function mapDispatchToProps (dispatch) {
  return {
    goToAllCommunities: () => dispatch(push(allCommunitiesUrl()))
  }
}

export default connect(null, mapDispatchToProps)
