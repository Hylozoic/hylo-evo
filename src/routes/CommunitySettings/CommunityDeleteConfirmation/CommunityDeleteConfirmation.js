import React from 'react'
import FullPageModal from 'routes/FullPageModal'

export default function CommunityDeleteConfirmation ({ goToAllCommunities }) {
  return <FullPageModal
    content={<div style={{textAlign: 'center'}}>
      Your community has been deleted.
    </div>}
    onClose={goToAllCommunities}
  />
}
