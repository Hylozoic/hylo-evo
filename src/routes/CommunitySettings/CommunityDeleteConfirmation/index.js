import CommunityDeleteConfirmation from './CommunityDeleteConfirmation'
import connector from './CommunityDeleteConfirmation.connector'
export default connector(CommunityDeleteConfirmation)
