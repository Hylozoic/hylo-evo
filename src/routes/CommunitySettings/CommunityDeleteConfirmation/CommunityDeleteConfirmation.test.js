import CommunityDeleteConfirmation from './CommunityDeleteConfirmation'
import { shallow } from 'enzyme'
import React from 'react'

it('renders correctly', () => {
  const wrapper = shallow(<CommunityDeleteConfirmation />)
  expect(wrapper).toMatchSnapshot()
})
