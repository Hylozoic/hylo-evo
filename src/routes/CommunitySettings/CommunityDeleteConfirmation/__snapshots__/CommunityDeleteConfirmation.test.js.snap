// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<withRouter(Connect(FullPageModal))
  content={
    <div
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      Your community has been deleted.
    </div>
  }
/>
`;
